### ISSUE TITLE
Chatwoot Dialogflow integration - using region other than global

### ISSUE THREAD

**Issue by SebastianBitsch** [chatwoot/chatwoot#4129]
**Describe the bug**

When trying to link my Dialogflow-agent to Chatwoot using the native integration (following the steps at: https://www.chatwoot.com/docs/product/channels/live-chat/integrations/dialogflow),
I find that i can connect to an agent which resides in the dialogflow region 'Global' without any issues whatsoever, but when i create an identical agent in another region (eu-w1 for instance), the connection fails.  

**To Reproduce**

1. Create a Chatwoot setup
2. Setup a dialogflow agent in the global region
3. Link the two using a service account as described in https://www.chatwoot.com/docs/product/channels/live-chat/integrations/dialogflow
4. Then do step 3 again for an agent in the 'eu-w1' region
5. See that the global agent works, but the one in eu doesnt.

**Expected behavior**

The agent with dialogflow region set to eu-w1 should connect to chatwoot without issues

**Server logs**

No server logs in dialogflow for the eu-w1 agent

Share the server logs to debug the issue further

**Environment**
Self hosted on Linux VM

**Additional context**
Please let me know if you need any additional logs, info etc.

Thanks

**Comment by ChrisvanChip** [comment]
Can confirm that this issue exists. Support for regions would be beneficial for GDPR compliance.

**Comment by SebastianBitsch** [comment]
@ChrisvanChip i agree, it is pretty problematic for metting gpdr compliance. Do you know of any good workaround?

**Comment by pranavrajs** [comment]
@SebastianBitsch Not sure, what is missing here. I will test it out and post the results.

**Comment by SebastianBitsch** [comment]
Hi @pranavrajs , any updates on this?

**Comment by pranavrajs** [comment]
@SebastianBitsch Sorry I couldn't investigate this yet, will prioritize for the next sprint.

**Comment by JarydX** [comment]
This problem might happen because of different API Endpoints for all regions? https://cloud.google.com/dialogflow/es/docs/reference/rest/v2-overview#service-endpoint

However, i can reproduce this issue. No other region than global is working right now.

**Comment by clstroem** [comment]
Hi @pranavrajs 
Any updates on this issue? 
Thanks😊

**Comment by hheexx** [comment]
please add this info to documentation if it's not ready yet, i just lost 2 days....

**Comment by des79** [comment]
Hi there, there is some updates on this? 

I've been trying to configure dialogflow for days without success but from what I understand here I can't use the regional one but I have to use the global one?

It's correct?

Thank you
